modules/eks/echo-server/charts/echo-server/templates/ingress.yaml

```diff
@@ -9,20 +9,10 @@ metadata:
   name: {{ $fullName }}
   annotations:
     {{- if eq (printf "%v" .Values.ingress.nginx.enabled) "true" }}
-    kubernetes.io/ingress.class: {{ .Values.ingress.nginx.class }}
       {{- if (index .Values.ingress.nginx "tls_certificate_cluster_issuer") }}
     cert-manager.io/cluster-issuer: {{ .Values.ingress.nginx.tls_certificate_cluster_issuer }}
       {{- end }}
     {{- else if eq (printf "%v" .Values.ingress.alb.enabled) "true" }}
-    kubernetes.io/ingress.class: {{ .Values.ingress.alb.class }}
-      {{- if not .Values.ingress.alb.group_name }}
-    alb.ingress.kubernetes.io/load-balancer-name: {{ index .Values.ingress.alb "load_balancer_name" | default "k8s-common" }}
-      {{- end }}
-    alb.ingress.kubernetes.io/group.name: {{ index .Values.ingress.alb "group_name" | default "common" }}
```

Comment:
this allows us to test the alb-ingress-controller-group logs being forwarded to datadog

Reply:
But it also can cause the whole thing to be destroyed when `echo-server` is removed. We do not want that to be default behavior. Any such configuration should be done explicitly outside of `echo-server`, on the default Ingress.